packages/vue/src/components/molecules/SfDropdown/SfDropdown.vue

```diff
@@ -1,6 +1,11 @@
 <template>
   <div class="sf-dropdown">
     <SfOverlay :visible="isOpen" class="sf-dropdown__overlay" />
+    <slot>
```

Comment:
Imho if we have a default opener it should be a default slot. default slot should be for a content, and for opener (as its optional) we should have it being a named slot like "opener"

Reply:
Ok, I'll fix that.